### DISCUSSION TITLE
vercel pro trial for default team - does it include myself

### DISCUSSION THREAD

**Discussion by antlossway** [vercel/community#5829] [Help] [answered]
### Summary

I just click to try pro, but it default to apply to "My Team", which only including me. 
But I suspect my existing projects are not included in the pro trial, because immediately the interface ask me to add a new project.
Can anyone tell me if existing project created under "Hobby" account are benefiting from the pro trial which is applied on "My Team" ?

### Steps to Reproduce

opt to experience pro trial

**Comment by swarnava** [comment, marked as answer]
Hi there,

You need to transfer your Hobby project to Pro team scope in order to get Pro trial benefit: https://vercel.com/docs/projects/overview#transferring-a-project. It doesn't happen automatically.
